### PR TITLE
Fix golint in staging/src/k8s.io/apiserver/plugin/pkg/authenticator/password/passwordfile

### PR DESCRIPTION
**Pull request by Asalle**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
fixes go lint errors

**Which issue(s) this PR fixes**:
xref #68026

**Special notes for your reviewer**:
SIG: K8s-Infra

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
```docs

```

### CODE REVIEW

**Comment by Asalle** [comment]
/assign liggitt
hey @liggitt, this is a very small pr that fixes golint for the package `passwordfile`. CLA is on the way.

**Comment by liggitt** [comment]
thanks, can you also remove the package from the list of lint failures?

**Comment by Asalle** [comment]
Yes, will do!

File: staging/src/k8s.io/apiserver/plugin/pkg/authenticator/password/passwordfile/passwordfile.go [hunk 1]
```diff
@@ -81,6 +82,7 @@ func NewCSV(path string) (*PasswordAuthenticator, error) {
 	return &PasswordAuthenticator{users}, nil
 }
 
+// AuthenticatePassword returns user info if authentication is successful, nil otherwise
```
Comment (jpbetz) [inline staging/src/k8s.io/apiserver/plugin/pkg/authenticator/password/passwordfile/passwordfile.go]: Document bool response as well? (error is more obvious, although, curiously, I don't see error ever returned in the implementation. Does this implement an interface?).

Comment (Asalle) [reply staging/src/k8s.io/apiserver/plugin/pkg/authenticator/password/passwordfile/passwordfile.go]: it does implement an interface from `staging/src/k8s.io/apiserver/pkg/authentication/authenticator/interfaces.go`. Strangely `bool` is not documented there as well.

**Pull request merged**